src/asset/js/clear.js

```diff
@@ -0,0 +1,13 @@
+/** @format */
+/* eslint-disable */
+import { showList } from './showList';
+/* eslint-disable */
+
+const clear = (todoTask) => {
+  todoTask = todoTask.filter((todo) => todo.completed !== true);
+  localStorage.setItem('todoTask', JSON.stringify(todoTask));
+  console.log(todoTask);
+  showList(todoTask);
+};
```

Comment:
- [ ] Kindly consider updating this function making sure that it gets data from the local storage and filters the one which their is complete is not equal to true and make sure that after filtering only the one with completed which is equal to false is the only one set to the local storage because currently after clearing all completed the local storage still keeps the todos which are completed 
- 
<img width="1409" alt="Screen Shot 2023-05-12 at 14 27 37" src="https://github.com/ghermaico135/todo-list/assets/60146030/da092b49-17b2-4d55-b9f1-86d92e4a67ec">


doing so will help you to meet the below requirement 
> The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored.